src/styles/stars.scss

```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-repeat: no-repeat;
+    margin-right: 4px;
+    background-position: center;
+    background-image: url(/src/images/star.svg);
+  }
+
+  &--4 &__star:nth-child(-n + 4) {
+    background-image: url(/src/images/star-active.svg);
+  }
```

Comment:
This code will only work for 4 stars. What if you need to reuse this code for a different number of active stars? It would be better to make this more flexible. You could use a loop to generate the classes for the different number of active stars. For example, you can use the `@for` directive in SCSS to achieve this.